scripts/android/build_all.sh

```diff
@@ -12,11 +12,11 @@ mkdir -p build
 
 PLUGINS_DIR=../../crypto_plugins
 
-(cd "${PLUGINS_DIR}"/flutter_liblelantus/scripts/android && ./build_all.sh ) &
-(cd "${PLUGINS_DIR}"/flutter_libepiccash/scripts/android && ./build_all.sh )  &
-(cd "${PLUGINS_DIR}"/flutter_libmonero/scripts/android/ && ./build_all.sh  ) &&
-set_rust_to_1720 &&
-(cd "${PLUGINS_DIR}"/frostdart/scripts/android && ./build_all.sh ) &
+(cd "${PLUGINS_DIR}"/flutter_liblelantus/scripts/android && ./build_all.sh )
+(cd "${PLUGINS_DIR}"/flutter_libepiccash/scripts/android && ./install_ndk.sh && ./build_openssl.sh && ./build_all.sh )
```

Comment:
libepiccash's build all script runs install ndk and openssl (or at least it did last I checked)

Reply:
Yes it does.  Resolved by ce17fed2bdc21147ed21164d505dd01d1feac2fbe4420f8c561ba6cd0da91607